doc/developer/bmp.rst

```diff
@@ -4,46 +4,37 @@
 BMP
 ***
 
-RFC 7854
-========
+RFC 7854: BMP Adj-RIB-In
+========================
 Missing features (non exhaustive):
-  - Per-Peer Header
-
-    - Peer Type Flag
-    - Peer Distingsher
-
-  - Peer Up
-
+  - Peer Down
     - Reason codes (according to TODO comments in code)
 
-Peer Type Flag and Peer Distinguisher can be implemented easily using RFC 9069's base code.
-
-RFC 9069
-========
+RFC 9069: BMP Local-RIB
+=======================
 Everything that isn't listed here is implemented and should be working.
 Missing features (should be exhaustive):
 
-- Per-Peer Header
-
-  - Timestamp
-
-    - set to 0
-    - value is now saved `struct bgp_path_info -> locrib_uptime`
-    - needs testing
-
-- Peer Up/Down
-
-  - VRF/Table Name TLV
+- Peer Down Only
+  - Reason code (bc not supported in RFC 7854 either)
 
-    - code for TLV exists
-    - need better RFC understanding
+RFC8671: BMP Adj-RIB-Out
+========================
+Adj-RIB-Out pre-policy uses tricks to work because we soft-reconfiguration outbound does not exist.
```

Comment:
the "we" is superfluous here.
i mean in frrouting we don't have an option to retain the state of the adj-rib-out pre-policy (soft-reconfiguration outobund) like we have for the adj-rib-in pre-policy (soft-reconfiguration inbound)

Reply:
then probably change it to "fr/r" or some such?

Reply:
@riw777 sorry i don't understand what you mean by "fr/r"?

Reply:
fr routing ...

Reply:
i'm really sorry but i don't understand where do you want me to add "frrouting"? 
to me this should change from 
```
RFC8671: BMP Adj-RIB-Out
========================
Adj-RIB-Out pre-policy uses tricks to work because we soft-reconfiguration outbound does not exist.
```

to
```
RFC8671: BMP Adj-RIB-Out
========================
Adj-RIB-Out pre-policy monitoring uses tricks to work because soft-reconfiguration outbound does not exist.
```